pkgs/applications/networking/cluster/cmctl/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildGo117Module, fetchFromGitHub, installShellFiles }:
+
+buildGo117Module rec {
+  pname = "cmctl";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "jetstack";
+    repo = "cert-manager";
+    rev = "v${version}";
+    sha256 = "sha256-3akDm3mAp10ilAz6oAMblXzBpU0BGMxzOTqHYJvg9AY=";
+  };
+
+  vendorSha256 = "sha256-4zhdpedOmLl/i1G0QCto4ACxguWRZLzOm5HfMBMtvPY=";
+
+  subPackages = [ "cmd/ctl" ];
+
+  ldflags = [ "-s" "-w" ];
```

Comment:
Does `registerCompletion` that was being done by the `substituteInPlace` still need to be added to `ldflags`?

Reply:
When I first packaged package it was necessary the monkey patch. But someone made a patch since then. It is okay now.